### ISSUE TITLE
Specified Icon font cdn not used

### ISSUE THREAD

**Issue by pierresaid** [userquin/vuetify-nuxt-module#59]
It seems the cdn option to override the default font icon cdn is not respected
https://vuetify-nuxt-module.netlify.app/guide/icons/font-icons.html#font-icons

```js
import { defineNuxtConfig } from 'nuxt/config'

export default defineNuxtConfig({
  modules: ['vuetify-nuxt-module', '@pinia/nuxt'],
  vuetify: {
    vuetifyOptions: {
      icons: {
        defaultSet: 'mdi',
        sets: [{
          name: 'mdi',
          cdn: 'https://cdnjs.cloudflare.com/ajax/libs/MaterialDesign-Webfont/7.2.96/css/materialdesignicons.min.css'
        }]
      }
    }
  }
})
```

but it still tries the default cdn : 
![image](https://github.com/userquin/vuetify-nuxt-module/assets/32095218/50b1ff7f-95c9-491c-abcf-448c864daecc)

(in my case cdn.jsdeliver is blocked)

**Comment by userquin** [comment]
@pierresaid  the cdn entry being ignored, it should be fixed in `0.4.12` version:

https://stackblitz.com/edit/nuxt-starter-rm5zev?file=package.json,nuxt.config.ts,app.vue

![imagen](https://github.com/userquin/vuetify-nuxt-module/assets/6311119/23c00a87-cf06-4f48-9577-c8d1af902e0d)

**Comment by pierresaid** [comment]
It works thanks !
